src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/ResultPropertiesTests.cs

```diff
@@ -368,7 +368,7 @@ void Test()
             });
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/58198")]
```

Comment:
I believe this can be unskipped because of #58203
```suggestion
        [Fact]
```